### DISCUSSION TITLE
Updating/Adding Types

### DISCUSSION THREAD

**Discussion by Grateful-Viv** [code4recovery/12-step-meeting-list#923] [Q&A] [answered]
I  would like to change the description for "Outdoor Meeting" to simply "Outdoor" since when it is selected from the dropdown, it displays as "Outdoor Meeting Meetings" and I think it looks weird.

![Outdoor-Meeting-Meetings](https://user-images.githubusercontent.com/62247421/190265848-a2031026-4ba8-4d18-9e18-237ba0e234dc.png)

To do this would I add the following to my child's functions.php?

```
if (function_exists('tsml_custom_types')) {
		tsml_custom_types(array(
			'OUT' => 'Outdoor',
		));
	}
```
And if I want to add a Promises type, how would I do this? Could it be on the same line such as 

```
if (function_exists('tsml_custom_types')) {
		tsml_custom_types(array(
			'OUT' => 'Outdoor', 'PRO' => 'Promises',
	        ));
	}
```
 Or something else? 

Finally, where can I see the list of types in use so I don't accidently replace an existing type?

**Comment by joshreisner** [comment, marked as answer]
Yes that code looks good. I might put each type on a different line, but it should work either way so long as there's a comma separating.

Full list of types: https://github.com/code4recovery/spec#meeting-types

Looks like TSML has a bug and `Outdoor Meeting` should just be `Outdoor` (among other tweaks) - I've created a ticket: https://github.com/code4recovery/12-step-meeting-list/issues/924

**Reply by Grateful-Viv** [reply]
Thanks @joshreisner. I’ll just add the Promises type if future updates will change Outdoor Meeting to Outdoor. 

Is my understanding correct that any new types I add won’t be reflected in the meeting guide app?

**Reply by joshreisner** [reply]
That's right
